pkg/apis/operators/v1/zz_generated.deepcopy.go

```diff
@@ -1,21 +1,5 @@
 // +build !ignore_autogenerated
 
-/*
```

Comment:
Are you sure the copyright got removed? Please double check this.

Reply:
I thought this was weird as well - it is removed by the `operator-sdk generate k8s` command. I verified by running said command in master branch and seeing the same change:
```
operator-sdk generate k8s
INFO[0000] Running deepcopy code-generation for Custom Resource group versions: [operators:[v1], ] 
giINFO[0001] Code-generation complete.                    
t(master → origin {7} U:1 ✗) operator-marketplace git status
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   pkg/apis/operators/v1/zz_generated.deepcopy.go

no changes added to commit (use "git add" and/or "git commit -a")
(master → origin {7} U:1 ✗) operator-marketplace git diff
diff --git a/pkg/apis/operators/v1/zz_generated.deepcopy.go b/pkg/apis/operators/v1/zz_generated.deepcopy.go
index 46bcc2d0..4494bc02 100644
--- a/pkg/apis/operators/v1/zz_generated.deepcopy.go
+++ b/pkg/apis/operators/v1/zz_generated.deepcopy.go
@@ -1,21 +1,5 @@
 // +build !ignore_autogenerated
 
-/*
-Copyright The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
 // Code generated by deepcopy-gen. DO NOT EDIT.
 
 package v1

``

Reply:
The copyright is missing because the SDK introduced some boilerplate code, I will add a commit to this PR that adds it.